### ISSUE TITLE
Bookmarks containing subdomains are reset, when the corresponding second level domain bookmark ist added.

### ISSUE THREAD

**Issue by bpcurse** [theScrabi/OCBookmarks#14]
Android 6.0.1, Bookmarks 1.2, Nextcloud Server 12.0.4

Steps to recreate:
1. Create bookmark - https://drive.google.com
2. Create bookmark - https://groups.google.com
3. Create bookmark - https://google.com

Expected behaviour:
Bookmarks are created independently and are not touched when the "pure" SLD bookmark is added in step 3.

Actual behaviour:
After step 3 the bookmarks of step 1 & 2 are changed to https://google.com

I could not recreate this in the web app. It only happens if you use the android app.

**Comment by theScrabi** [comment]
does this error exist when you try to add this using the owncloud/nextcloud app instead of the android app?

**Comment by bpcurse** [comment]
It works fine with the nextcloud bookmarks web app. Seems to be android app specific.

**Comment by theScrabi** [comment]
I see.

**Comment by theScrabi** [comment]
confirmed.

**Comment by theScrabi** [comment]
closing in favour of https://gitlab.com/derSchabi/OCBookmarks/issues/14
